src/gpio.rs

```diff
@@ -699,6 +695,19 @@ macro_rules! gpio {
 
                     #[cfg(feature = "unproven")]
                     impl<MODE> toggleable::Default for $PXi<Output<MODE>> {}
+
+                    impl<MODE> InputPin for $PXi<Output<MODE>> {
+                    type Error = ();
+
+                    fn is_high(&self) -> Result<bool, Self::Error> {
+                        self.is_low().map(|v| !v)
+                    }
+
+                    fn is_low(&self) -> Result<bool, Self::Error> {
+                        // NOTE(unsafe) atomic read with no side effects
+                        Ok(unsafe { (*$GPIOX::ptr()).idr.read().bits() & (1 << $i) == 0 })
+                    }
+                }
                 )+
```

Comment:
```suggestion
                    type Error = ();

                    fn is_high(&self) -> Result<bool, Self::Error> {
                        self.is_low().map(|v| !v)
                    }

                    fn is_low(&self) -> Result<bool, Self::Error> {
                        // NOTE(unsafe) atomic read with no side effects
                        Ok(unsafe { (*$GPIOX::ptr()).idr.read().bits() & (1 << $i) == 0 })
                    }
                }
                )+
```

Indentation is wrong. I wonder why rustfmt did not complaint :thinking: